src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/Departures/Documents/PreviousStepDef.scala

```diff
@@ -17,6 +17,7 @@
 package uk.gov.hmrc.test.ui.cucumber.stepdefs.Departures.Documents
 
 import uk.gov.hmrc.test.ui.cucumber.stepdefs.BaseStepDef
+import uk.gov.hmrc.test.ui.pages.Departures.Documents.Documents.{AddAdditionalInformationPage, DocumentEnterAdditionalInfoPage}
```

Comment:
Just a suggestion here. You can use '._' instead of listing the individual pages in the import statement.

Reply:
ok